src/js/components/forms/Dropdown.jsx

```diff
@@ -56,6 +56,18 @@ export class Dropdown extends React.Component {
         return this.state.selectedOption;
     }
 
+    selectOption(option) {
```

Comment:
Just interested why a method that requires a ref rather than a prop? I'm worried we might be fighting against the "react way" a bit too much when we do stuff like this. Maybe we should separate out the job of maintaining "selected item" state into (the|a) container component..... Aaaaand now I'm rambling...